kaniko/code/buildpackconfig/buildPackConfig.go

```diff
@@ -92,29 +83,13 @@ func (b *BuildPackConfig) InitDefaults() {
 		})
 	}
 
-	logrus.Debug("Checking if CNB_* env var have been declared ...")
-	b.CnbEnvVars = util.GetCNBEnvVar()
-	logrus.Debugf("CNB ENV var is: %s", b.CnbEnvVars)
-
-	// Convert the CNB ENV vars to Kaniko BuildArgs
-	for k, v := range b.CnbEnvVars {
-		logrus.Debugf("CNB env key: %s, value: %s", k, v)
-		arg := k + "=" + v
-		b.BuildArgs = append(b.BuildArgs, arg)
-	}
-
-	// setup the path to access the Dockerfile within the workspace dir
-	dockerFilePath := b.WorkspaceDir + "/" + b.DockerFileName
```

Comment:
Same comment as above

Reply:
README should also remove this section explaining how to pass parameters using TOML Dockerfiles and not ENV Vars  ;-)